#!/usr/bin/env python
# field of view configuration

PACKAGE="shape_finder"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("leafsize", double_t, 1, "Leafsize", 0.02, 0.002, 0.3)
gen.add("overlap_threshold", double_t, 1, "Overlap_threshold", 0.6, 0.001, 1)
gen.add("big_plane_size", int_t, 1, "big_plane_size", 1000, 10, 10000)
gen.add("min_cloud_size", int_t, 1, "min_cloud_size", 100, 10, 10000)

exit(gen.generate(PACKAGE, "shape_finder", "shape_finder_node"))
